packages/runner-shared/src/store.ts

```diff
@@ -0,0 +1,39 @@
+import { action, observable } from 'mobx'
+import { nanoid } from 'nanoid'
+
```

Comment:
The purpose of this is to share the common methods; eventually, this store should have more code, and the individual runners have less.